src/main/java/launcher/Launcher.java

```diff
@@ -28,6 +28,7 @@
      */
     public static void main(final String[] args) throws FileNotFoundException {
         Settings.initialize("game.properties");
```

Comment:
For naming conventions maybe change initialize to initializeProperties (to align with initializeHighscores)
<naming